src/applications/mhv-medical-records/components/LabsAndTests/RadiologyDetails.jsx

```diff
@@ -451,29 +448,28 @@ ${record.results}`;
         >
           {record.imagingProvider}
         </p>
-        {!phase0p5Flag && (
-          <>
-            <h3 className="vads-u-font-size--md vads-u-font-family--sans no-print">
-              Images
-            </h3>
-            <p data-testid="radiology-image" className="no-print">
-              Images are not yet available in this new medical records tool. To
-              get images, you’ll need to request them in the previous version of
-              medical records on the My HealtheVet website.
-            </p>
-            <va-link
-              href={mhvUrl(
-                isAuthenticatedWithSSOe(fullState),
-                'va-medical-images-and-reports',
-              )}
-              text="Request images on the My HealtheVet website"
-              data-testid="radiology-images-link"
-              onClick={() => {
-                sendDataDogAction('Request images on MHV');
-              }}
-            />
-          </>
-        )}
+
+        <>
+          <h3 className="vads-u-font-size--md vads-u-font-family--sans no-print">
+            Images
+          </h3>
+          <p data-testid="radiology-image" className="no-print">
+            Images are not yet available in this new medical records tool. To
+            get images, you’ll need to request them in the previous version of
+            medical records on the My HealtheVet website.
+          </p>
+          <va-link
+            href={mhvUrl(
+              isAuthenticatedWithSSOe(fullState),
+              'va-medical-images-and-reports',
+            )}
+            text="Request images on the My HealtheVet website"
+            data-testid="radiology-images-link"
+            onClick={() => {
+              sendDataDogAction('Request images on MHV');
+            }}
+          />
+        </>
```

Comment:
For instances of `!phase0p5Flag`, we should remove the entire condition block.

Reply:
Done.